pkg/web/client.go

```diff
@@ -102,19 +108,19 @@ func (c *Client) PostNewVersion(ctx context.Context, image string, weights []Fil
 func (c *Client) versionFromManifest(image string, weights []File, files []File) (*Version, error) {
 	manifest, err := c.dockerCommand.Inspect(image)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to inspect docker image: %v", err)
```

Comment:
make sure to use `fmt.Errorf` with `%w` instead of `%v` so that the error is wrapped instead of stringified